app/components/HowTo/index.js

```diff
@@ -297,8 +297,9 @@ const HowTo = () => {
                 and select <b>Withdraw Funds</b>.
                 <ul>
                   <li>
-                    There will be a <b>10% service fee</b> on withdrawals, as
-                    well as applicable payment processing fees.
+                    There will be a <b>3% + $0.30 standard transaction fee</b>{' '}
```

Comment:
edited text to: 
`There will be a 3% + $0.30 standard transaction fee on withdrawals to cover payment processing`